docs/admin/authentication.md

```diff
@@ -367,6 +367,12 @@ option to the API server during startup. The plugin is implemented in
 `plugin/pkg/auth/authenticator/password/keystone/keystone.go` and currently uses
 basic auth to verify used by username and password.
 
+If you have configured self-signed certificates for Keystone server,
+`--experimental-keystone-ca-file=SOMEFIlE` option may be needed during the startup
```

Comment:
"If you have configured self-signed certificates for the Keystone server, you may need to set the --experimental-keystone-ca-file=SOMEFILE` option when starting the Kubernetes API server. If you set the option, the Keystone server's certificate is verified by one of the authorities in the `experimental-keystone-ca-file`. Otherwise, the certificate is verified by the host's root Certificate Authority."